execution_nodes/execute_node.js

```diff
@@ -36,6 +36,10 @@ app.use(express.static(__dirname + '/public'));
 app.use(bodyParser.urlencoded({extended: true}));
```

Comment:
This is a misplaced comment. Functional tests may need to cover lines 151, 164, 165 of execution_nodes.js.